Cubical/Experiments/Poset.agda

```diff
@@ -0,0 +1,326 @@
+{-
+ This an earlier formalization of posets using the old SIP instead of DURGs
```

Comment:
Please add this to `Cubical.Experiments.Everything`

Reply:
Done